src/me/ryanhamshire/PopulationDensity/EntityEventHandler.java

```diff
@@ -148,14 +140,14 @@ public void onItemDespawn (ItemDespawnEvent event)
 		
 		for(int i = 0; i < neighbors.length; i++)
 		{
-			if(neighbors[i].getType() == Material.SAPLING || neighbors[i].getType() == Material.LOG) return;
+			if((neighbors[i].getType() == Material.OAK_SAPLING) || (neighbors[i].getType() == Material.SPRUCE_SAPLING) || (neighbors[i].getType() == Material.BIRCH_SAPLING) || (neighbors[i].getType() == Material.JUNGLE_SAPLING) || (neighbors[i].getType() == Material.ACACIA_SAPLING) || (neighbors[i].getType() == Material.DARK_OAK_SAPLING) || (neighbors[i].getType() == Material.OAK_LOG) || (neighbors[i].getType() == Material.SPRUCE_LOG) || (neighbors[i].getType() == Material.BIRCH_LOG) || (neighbors[i].getType() == Material.JUNGLE_LOG) || (neighbors[i].getType() == Material.ACACIA_LOG) || (neighbors[i].getType() == Material.DARK_OAK_LOG)) return;
```

Comment:
I think a `switch` would be more efficient and clearer.

```javascript
switch (neighbors[i].getType()) {
	case Material.OAK_SAPLING:
	case Material.SPRUCE_SAPLING:
	case Material.BIRCH_SAPLING:
	case Material.JUNGLE_SAPLING:
	case Material.ACACIA_SAPLING:
	case Material.DARK_OAK_SAPLING:
	case Material.OAK_LOG:
	case Material.SPRUCE_LOG:
	case Material.BIRCH_LOG:
	case Material.JUNGLE_LOG:
	case Material.ACACIA_LOG:
	case Material.DARK_OAK_LOG:
		return;
}
```

However, because saplings are used more than once, and it might also be the case with logs, I think it wouldn't hurt to add a global constant array that contains the various types of saplings and logs and use it multiple times.

```javascript
const material_saplings = [
	Material.OAK_SAPLING,
	Material.SPRUCE_SAPLING,
	Material.BIRCH_SAPLING,
	Material.JUNGLE_SAPLING,
	Material.ACACIA_SAPLING,
	Material.DARK_OAK_SAPLING
];

const material_logs = [
	Material.OAK_LOG,
	Material.SPRUCE_LOG,
	Material.BIRCH_LOG,
	Material.JUNGLE_LOG,
	Material.ACACIA_LOG,
	Material.DARK_OAK_LOG
];
```

It then would be possible to use it like so : 

```javascript
const material = neighbors[i].getType();
if (material_saplings.indexOf(material) == -1 && material_logs.indexOf(material) == -1) {
	// material is not in material_saplings nor material_logs.
}
```